react-charts/bar-chart/step5.md

```diff
@@ -1,34 +1,68 @@
-Now we'll add tooltips to the bar chart...
+Add axis labels to the bar chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
-
-2) Locate the code that looks like the following:
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
 <pre class="file">
 &lt;Chart
+  containerComponent={&lt;ChartVoronoiContainer
+  labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
   domainPadding={{ x: [30, 25] }}
   height={250}
-  width={600}
-&gt;
+  width={600}&gt;
 </pre>

```

Comment:
This was split into multiple steps for the area chart tutorial. I believe we want to mimic that here as well.

Reply:
not sure what this means

Reply:
^ looks like you split it up

Reply:
ok, then